modules/database_layer/variables.tf

```diff
@@ -86,7 +86,41 @@ variable "db_port_num" {
 }
 
 #--------------------------------------------------------------
-# Other Module Vars
+# AWS SSM Configuration
+#--------------------------------------------------------------
+
+variable "ssm_prefix" {
+  description = "If set, saves db parameters to the parameter store with the given prefix"
+  default     = ""
+}
+
+variable "ssm_overwrite" {
+  description = "If true any existing parameters will be overwritten"
+  default     = "false"
```

Comment:
Now that SSM has versioning, should this be true?

Reply:
I was hesitant because I haven't set up chamber yet, but will update it for the pattern :)